classroom/snippets/createCourse.go

```diff
@@ -46,6 +47,19 @@ func createCourse(client *http.Client) {
 	if err != nil {
 		log.Fatalf("Course unable to be created %v", err)
 	}
-	// [END classroom_create_course]
 	fmt.Printf("Created course: %v", course.Id)
+	// [END classroom_create_course]
```

Comment:
Per earlier guidance on snippets, expand the area in the region tags to include imports & auth.